api/src/services/dwc-service.ts

```diff
@@ -90,4 +70,60 @@ export class DwCService extends DBService {
     // Apply patch operations
     return jsonpatch.applyPatch(jsonObject, patchOperations).newDocument;
   }
+
+  async decorateDWCASourceData(occurrenceSubmissionId: number): Promise<boolean> {
+    const occurrenceService = new OccurrenceService(this.connection);
+
+    const submission = await occurrenceService.getOccurrenceSubmission(occurrenceSubmissionId);
+    const jsonObject = submission.darwin_core_source;
+
+    const pathsToPatch = JSONPath({
+      path: '$..[verbatimCoordinates]^',
+      json: jsonObject,
+      resultType: 'all'
+    });
+
+    const patchOperations: Operation[] = [];
+
+    await Promise.all(
+      pathsToPatch.map(async (item: any) => {
+        // eslint-disable-next-line no-prototype-builtins
+        if (item.value.hasOwnProperty('decimalLatitude') && item.value.hasOwnProperty('decimalLongitude')) {
```

Comment:
I think the linter makes a valid case: https://eslint.org/docs/latest/rules/no-prototype-builtins

You can ameliorate the complaint by calling `Object.prototype.hasOwnProperty`:

```suggestion
        if (
          Object.prototype.hasOwnProperty.call(item.value, 'decimalLatitude') &&
          Object.prototype.hasOwnProperty.call(item.value, 'decimalLongitude')
        ) {
```

Reply:
Agreed. Lets prefer avoiding disabling linter/formatter rules whenever possible.